recipes/example/orange3-fairness/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "Orange3-Fairness" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18f85525bf1a4b1cf6286a92051b6053ad745826311c037cd40482411b95824b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - aif360 ==0.5.0
+    - numpy 1.23.*
+    - orange3
+    - pip
+    - python
+    - setuptools
+  run:
+    - aif360 ==0.5.0
+    - numpy 1.23.*
+    - orange3
+    - python
+    - setuptools
+
+test:
+  imports:
+    - orangecontrib
+    - orangecontrib.fairness
+    - orangecontrib.fairness.evaluation
+    - orangecontrib.fairness.modeling
+    - orangecontrib.fairness.widgets
+    - orangecontrib.fairness.widgets.tests
+
+about:
+  home: https://orangedatamining.com/
+  license: GPL-3.0-only
+  license_family: GPL3
+  license_file: LICENSE
+  summary: "Orange3 add-on for fairness-aware machine learning."
+  doc_url: https://orange3-fairness.readthedocs.io/en/latest/
+  dev_url: https://github.com/biolab/orange3-fairness
+
+extra:
+  recipe-maintainers:
+    - zanmervic
```

```suggestion
    - zanmervic
    - primozgodec
```

It is good that each addon recipe is maintained by more people in case one is not available when it is needed. I am adding myself, but you can also add anyone that you think can help or is included in the project.